workflow/persist/sqldb/sqldb.go

```diff
@@ -85,6 +85,14 @@ func CreatePostGresDBSession(kubectlConfig kubernetes.Interface, namespace strin
 		Host:     postgresConfig.Host + ":" + postgresConfig.Port,
 		Database: postgresConfig.Database,
 	}
+
+	if postgresConfig.SSL {
+		var options := map[string]string{
+			"sslmode": "true"
+		}
+		settings.Options = options
+	}
```

Comment:
This code has language errors and won't compile. Please replace with:

```go
	if postgresConfig.SSL {
		options := map[string]string{
			"sslmode": "true",
		}
		settings.Options = options
	}
```

Reply:
@AntoineDao! Please fix @simster7 comments. PR looks good.

Reply:
Man I'm a bit embarrassed I didn't pick that up :see_no_evil:... I'll up my Go game before my next PR I promise!